tasks/input_sql_files.yml

```diff
@@ -36,6 +36,13 @@
     (__mssql_sqlcmd_login_cmd is none) or
     (__mssql_sqlcmd_login_cmd is not defined)
 
-- name: Input SQL files to MSSQL
+- name: Input SQL script files to MSSQL
   include_tasks: sqlcmd_input_file.yml
   loop: "{{ __mssql_sql_files_to_input }}"
+
+- name: Input SQL script contents to MSSQL
+  when: >-
```

Comment:
Why does the `when` check `mssql_pre_input_sql_content` and `mssql_post_input_sql_content` instead of just `__mssql_sql_content_to_input`?

Reply:
That was a bug, if the loop_var is empty it won't be processed within the loop so this `when` was not required

Reply:
Hm no, after all this is required, what you was saying was correct